supplementary/alica_msgs/package.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<package format="2">
+<package format="3">
```

Comment:
The reason I did it this way was to avoid duplicating the messages - if we split them up then all the .msg files would be duplicated, since nothing actually changed with how they work between ROS 1 and 2.